.gitignore

```diff
@@ -1,2 +1,4 @@
 target/
 **/*.rs.bk
+src/asset.in
+frontend/dist/
```

Comment:
If we update the assets and they're version controlled then the build script will work as it is now. It's not dependent on ember but it's still inflexible in a way. How about for now keeping the frontend/dist folder checked into git and then having the build script. I think that's a good medium until a more pluggable idea can be come up with.